docs/api.md

```diff
@@ -646,7 +646,7 @@ try {
 - `options` <[Object]> Set of configurable options to set on the browser. Can have the following fields:
   - `product` <[string]> Which browser to launch. At this time, this is either `chrome` or `firefox`. See also `PUPPETEER_PRODUCT`.
   - `ignoreHTTPSErrors` <[boolean]> Whether to ignore HTTPS errors during navigation. Defaults to `false`.
-  - `headless` <[boolean]> Whether to run browser in [headless mode](https://developers.google.com/web/updates/2017/04/headless-chrome). Defaults to `true` unless the `devtools` option is `true`.
+  - `headless` <[boolean]|"chrome"> Whether to run browser in [headless mode](https://developers.google.com/web/updates/2017/04/headless-chrome). Defaults to `true` unless the `devtools` option is `true`. "chrome" is a new experimental headless mode (you at your own risk).
```

Comment:
I actually wonder what is the difference between `chrome` headless and `true` headless. Is there a documentation on the difference? 🤔

If yes, can we link it in here?
If no, can we add a bit more explanation on the difference between these two modes to this documentation?

Reply:
I think the new mode is experimental for now so we don't want to advertise it too much. We can document it once it's more stable. Let's chat about the difference offline! For now we can just monitor and try to address CI failures to make it ready to be used.

Reply:
Makes sense, SGTM! 🥳